src/drm.cpp

```diff
@@ -1124,7 +1127,7 @@ drm_prepare_liftoff( struct drm_t *drm, const struct Composite_t *pComposite, co
 			if (g_bRotated) {
 				const int32_t x = crtcX;
 				const uint64_t w = crtcW;
-				crtcX = crtcY;
+				crtcX = crtcY + (g_nOutputHeight - crtcH);
```

Comment:
I've made a drawing, and I would've expected this formula to be `crtcX = outputHeight - crtcY - crtcH`.

Reply:
Thanks, was bit by a double negative as I thought crtcY being negated above made was making it negative, but it was already negative coming from steamcompmgr.